build.gradle.kts

```diff
@@ -341,12 +344,74 @@ tasks {
     return buildCodyDir
   }
 
+  fun copyProtocol(): Unit {
+    // This is just a temporary solution to make it easier to use certain generated protocol
+    // messages.
+    // Ultimately this should entirely be replaced by use of the generated protocol.
+    if (!isForceProtocolCopy) {
+      return
+    }
+    val fromEnvironmentVariable = System.getenv("CODY_DIR")
```

Comment:
You can use `downloadCody()` instead.
It uses cody from `CODY_DIR` if defined, if not then downloads cody specified by [cody.commit](https://github.com/sourcegraph/jetbrains/blob/main/gradle.properties#L27) (or uses one from cache, if exists).
This way it will also work without specifying `CODY_DIR`.